protocol/x/clob/types/mem_clob_keeper.go

```diff
@@ -57,6 +57,17 @@ type MemClobKeeper interface {
 		offchainUpdates *OffchainUpdates,
 		err error,
 	)
+	AddPreexistingStatefulOrder(
+		ctx sdk.Context,
+		order *Order,
+		blockHeight uint32,
```

Comment:
nit: can we update this to not take in a blockHeight? As far as I can tell the blockHeight isn't actually used in this code path. This is being passed in because the `PerformStatefulOrderValidation` function takes in a blockHeight but it is only used for short-term orders. The other place in the code where we call `AddPreexistingStatefulOrder` always passes in zero as blockHeight